app/components/notification-list-row/template.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="flex-container">
+    <div class="notification-title">New pending submissions</div>
+    {{#bs-dropdown as |dd|}}
+        {{#dd.button class="dropdown"}} {{t currentSettingLabel}} <span class="fa fa-caret-down p-l-xs"></span>{{/dd.button}}
```

Comment:
I believe you are referring to this button.

Reply:
Yup, that is the button I meant!